typings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Do we need this, given that TypeScript now supports the @types organization?

Reply:
No, please not! my bad.

Reply:
The typings.json helps consumers in some scenarios to get the typings of the aurelia-open-id-connect package. 
For the rest, use @types instead.